Systems/glass-effects.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<PropertyList>
+
+    <filter>
+        <name>splashX</name>
+        <type>gain</type>
+        <input>
+            <property>/fdm/jsbsim/glass-effects/splashx</property>
+        </input>
+        <output>
+            <property>/environment/aircraft-effects/splash-vector-x</property>
```

Comment:
I thought about that but I didn't think you could get out of /fdm/jsbsim/ space to /effects/ which is what the shader I looking for?
I'm looking for a way to override the shader but am getting conused.